tests/override_config_table/test_override_config_table_masic.py

```diff
@@ -0,0 +1,250 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import skip_release
+from tests.common.utilities import update_pfcwd_default_state
+from tests.common.config_reload import config_reload
+from utilities import backup_config, restore_config, get_running_config,\
+    reload_minigraph_with_golden_config, file_exists_on_dut
+
+NON_USER_CONFIG_TABLES = ["FLEX_COUNTER_TABLE"]
+NON_ASIC_CONFIG_TABLES = ["TACPLUS_SERVER", "FLEX_COUNTER_TABLE", "DEVICE_METADATA",
+                          "MGMT_VRF_CONFIG", "SYSLOG_SERVER", "DHCP_SERVER",
+                          "NTP_SERVER", "KUBERNETES_MASTER", "CONSOLE_PORT",
+                          "MUX_CABLE", "FG_NHG_MEMBER", "FG_NHG", "NEIGH",
+                          "SYSTEM_DEFAULTS"]
+GOLDEN_CONFIG = "/etc/sonic/golden_config_db.json"
+GOLDEN_CONFIG_BACKUP = "/etc/sonic/golden_config_db.json_before_override"
+CONFIG_DB = "/etc/sonic/config_db.json"
+CONFIG_DB_BACKUP = "/etc/sonic/config_db.json_before_override"
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t2', 't1'),
+    pytest.mark.disable_loganalyzer,
+]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def check_image_version(duthost):
+    """Skips this test if the SONiC image installed on DUT is older than 202111
+
+    Args:
+        duthost: DUT host object.
+
+    Returns:
+        None.
+    """
+    skip_release(duthost, ["201811", "201911", "202012", "202106", "202111"])
+
+
+@pytest.fixture(scope="module")
+def setup_env(duthost, tbinfo):
+    """
+    Setup/teardown
+    Args:
+        duthost: DUT.
+    """
+    topo_type = tbinfo["topo"]["type"]
+    if topo_type in ["m0", "mx"]:
+        original_pfcwd_value = update_pfcwd_default_state(duthost, "/etc/sonic/init_cfg.json", "disable")
+    # Backup configDB
+    for asic_id in duthost.get_frontend_asic_ids():
+        config = "/etc/sonic/config_db{}.json".format(asic_id)
+        config_backup = "/etc/sonic/config_db{}.json_before_override".format(asic_id)
+        backup_config(duthost, config, config_backup)
+    backup_config(duthost, CONFIG_DB, CONFIG_DB_BACKUP)
+    # Backup Golden Config if exists.
+    for asic_id in duthost.get_frontend_asic_ids():
+        golden_config = "/etc/sonic/golden_config_db{}.json".format(asic_id)
```

Comment:
From my understanding, we should always have one golden_config_db.json.
It covers all asic's config.

Reply:
hi @wen587 I'm on a multi-asic dut and there are golden config per asic and host
admin@xxx-1:/etc/sonic$ ls golden*
golden_config_db0.json  golden_config_db1.json  golden_config_db.json

Reply:
Hi @wenyiz2021 , that is strange...Let me start a teams chat on this.